argocd-clusterprofile-syncer/README.md

```diff
@@ -0,0 +1,81 @@
+# Argo CD ClusterProfile Syncer
+
+## Overview
+Argo CD ClusterProfile Syncer is a controller that syncs ClusterProfiles to Argo CD cluster secrets. It watches for changes in the cluster inventory and updates the Argo CD cluster secrets accordingly. For example, when a cluster is added to the inventory, the corresponding Argo CD cluster secret will be automatically generated. This allows Argo CD to seamlessly deploy and manage your applications to all clusters in the cluster inventory.
+
+## Install
+
+### Prerequisites
+- A hub cluster. For example, to create a GKE hub cluster, run:
+```shell
+gcloud container clusters create hub-cluster --region=${LOCATION?} \
+  --workload-pool=${PROJECT_ID?}.svc.id.goog
```

Comment:
```suggestion
  --workload-pool=${PROJECT_ID?}.svc.id.goog --enable-fleet
```

the hub cluster must be in the fleet so that you can then label the membership.